pavelib/paver_tests/utils.py

```diff
@@ -58,3 +61,52 @@ def info(self, message, *args):
             output = message
         if not output.startswith("--->"):
             self.messages.append(unicode(output))
+
+
+class CustomShMock(object):
```

Comment:
Why is this a class? The methods on it are just functions (they never use `self`).